source/develop/release-management/features/storage/backup-storage-domain.html.md

```diff
@@ -0,0 +1,248 @@
+---
+title: BackupStorageDomain
+category: feature
+authors: mlipchuk, shubham0d
+wiki_category: Feature|BackupStorageDomain
+wiki_title: Features/BackupStorageDomain
+wiki_revision_count: 0
+wiki_last_updated: 2017-24-03
+feature_name: Backup Storage Domain
+feature_modules: engine/vdsm
+feature_status: Yet to Impelement
+--- 
+
+
+# Backup Storage Domain
+
+This feature will add another functionality layer to the oVirt Disaster Recovery solution.
+
+
+## Owner
+
+* Maor Lipchuk <mlipchuk@redhat.com>
+* Shubham Dubey <sdubey504@gmail.com>
+
+
+## Current status
+
+* Yet to Implement
+
+
+## Summary
+
+Until now, one of the solutions to backup VMs and Templates in oVirt is to use an export storage domain.
+Export storage domain is a dedicated storage domain which is used to store and restore VMs and Templates.
+The drawback of Export storage domain is its two stage process. Transferring a VM required first copying it into the export storage domain and then into other storage domain to start using them. Since in background transfer from one domain to other take place using normal copy or dd utility(in case of disks) this process take a large time for a complete transfer. Hence, for a better backup experience we decided to create a whole dedicated storage domain to be used only for backup which obviously will call as backup storage domain.
+
+
+## GSOC
+
+This feature is being developed as part of the Google Summer of Code (GSOC).
+Google Summer of Code is a global program focused on introducing students to open source software
+development. Students work on a 3 month programming project with an open source organization
+during their break from university.<br />
+The oVirt organization has chosen Shubham Dubey, a student from The LNM Institute of Information Technology from India, to work  on this project for the upcoming summer and Maor Lipchuk, a senior software developer in the oVirt storage team, to be the mentor.
+
+
+## Backup Storage Domain - Functionality
+
+* You can select any data storage domain as backup storage.
+* Once the storage domain is configured as backup the engine will block any running VMs or any changes that might be in the storage domain.
+* A backup storage domain can be detached and attached to a data center as every data storage domain.
+* Backup storage domain will be able to support unregistered VMs/Templates/Disks.
+* The backup indication will only be configured using the DataBase and will not be configured in the storage domain metadata - The user will be able to configure a storage domain as backup once a data storage domain is being imported or added to oVirt.
+* There is no obligation regarding data center version in oVirt to a backup storage domain. A backup storage domain can be part of every Data Center version.
+* A user can run a VM based on a Template which one of its disks is part of a backup storage domain while all the VM's disks are not part of the backup storage domain.
+
```

Comment:
Isn't a "backup storage domain" just a data storage domain with an extra property? If so, a lot of the points here are redundant.